pkgs/applications/misc/syncthingtray/default.nix

```
@@ -95,6 +96,8 @@ stdenv.mkDerivation (finalAttrs: {
     ++ lib.optionals (!webviewSupport) ["-DWEBVIEW_PROVIDER:STRING=none"]
   ;
 
+  qtWrapperArgs = [ "--prefix PATH : ${lib.makeBinPath [ xdg-utils ]}" ];
```

Thanks a lot for observing this! Could you please update to:

```suggestion
  qtWrapperArgs = [
    "--prefix PATH : ${lib.makeBinPath [
      xdg-utils
    ]}"
  ];
```

---

Could do, but IMO this makes it less readable for no real gain given there's only one package being added to the path. How about the following instead:

```nix
  qtWrapperArgs = [
    "--prefix PATH : ${lib.makeBinPath [ xdg-utils ]}"
  ];
```